roles/openshift_logging_defaults/defaults/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+openshift_logging_es5_techpreview: False
```

Comment:
so, we actually are going to be updating the logic of this. We will be replacing the ES 2.x stack with the 5.x stack and not support both installations at once with v3.11

Reply:
I'm hoping to ship this in 3.10.0 as we're getting some strong encouragement to do so.

Reply:
I'm not sure where your 'encouragement' is coming from, but ES5 is officially not a thing for 3.10, not even as techpreview.  It will be pushed out as part of 3.11

Reply:
The encouragement is to make oreg_url the only string that one needs to set in order to use an alternate registry, not directly related to ES 2.x vs 5.x.